### ISSUE TITLE
Storyshots gives "implicit any" / "can't find module" errors

### ISSUE THREAD

**Issue by su-narthur** [storybooks/storybook#1181]
I'm trying to set up Storyshots in my project. `Storyshots.test.ts` looks like this:

```js
import initStoryshots from '@storybook/addon-storyshots/'

initStoryshots();
```

When I run the tests, the test runner gives me this error:

>     Cannot find module '../../package' from 'node.js'
>       
>       at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:179:17)
>       at Object.<anonymous> (node_modules/babel-core/lib/api/node.js:60:16)

My IDE highlights the `import` statement in `Storyshots.test.ts` with this error:

> TS7016:Could not find a declaration file for module '@storybook/addon-storyshots/'. '[path to project]/node_modules/@storybook/addon-storyshots/dist/index.js' implicitly has an 'any' type.

Here are the versions I'm on:

```json
  "dependencies": {
    "@types/jest": "^19.2.3",
    "@types/node": "^7.0.18",
    "@types/react": "^15.0.24",
    "@types/react-dom": "^15.5.0",
    "@types/react-router-dom": "^4.0.4",
    "react": "^15.5.4",
    "react-dom": "^15.5.4",
    "react-relay": "^1.0.0",
    "react-router-dom": "^4.1.1",
    "styled-components": "^2.0.0",
    "ts-loader": "^2.1.0",
    "webpack": "^2.6.0"
  },
  "devDependencies": {
    "@storybook/addon-actions": "^3.0",
    "@storybook/addon-centered": "^3.0.0",
    "@storybook/addon-storyshots": "^3.0.0",
    "@storybook/react": "^3.0",
    "react-scripts-ts": "2.0.1",
    "react-test-renderer": "^15.5.4"
  },
```

How can I fix this?

**Comment by HBCharles** [comment]
What was your fix ? I'm also having the issue.

**Comment by mattleff** [comment]
@HBCharles There were several issues that we worked through. For reference we are using [react-scripts-ts](https://github.com/wmonk/create-react-app-typescript). 

> TS7016:Could not find a declaration file for module...

This is due to using `noImplicitAny` but not having types for Storyshots. Hopefully better types will come with #1166. Our solution for now is to have a `declarations.d.ts`:

```ts
declare module '@storybook/addon-storyshots'
```

> Cannot find module '../../package' from 'node.js'

This issue was the trickiest. Ultimately the fix I found is https://github.com/wmonk/create-react-app-typescript/pull/78. I'm using a fork until/if that pull is merged.

Hope this helps!
